70_FLIPFluids/app_resources/common.hlsl

```diff
@@ -0,0 +1,38 @@
+#ifndef _FLIP_EXAMPLE_COMMON_HLSL
+#define _FLIP_EXAMPLE_COMMON_HLSL
+
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+#define NUM_THREADS 128
+
+NBL_CONSTEXPR uint32_t WorkgroupSize = NUM_THREADS;
+NBL_CONSTEXPR float ratioFLIPPIC = 0.95;
+NBL_CONSTEXPR float deltaTime = 1.0f / 60.0f;
+NBL_CONSTEXPR float gravity = 15.0f;
+
+#ifdef __HLSL_VERSION
+
+static const float FLT_MIN = 1.175494351e-38;
+static const float FLT_MAX = 3.402823466e+38;
```

Comment:
slight nitpick, we have this https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/builtin/hlsl/limits.hlsl

because there's no constexpr functions in HLSL and constexpr float variables were introduced in C++20 the usage is
```cpp
numeric_limits<float>::min;
```

Reply:
still a todo

Reply:
unused variables